Makefile

```diff
@@ -86,10 +86,12 @@ ui-static: ## build static files for UI to be served by embedded API server
 
 .PHONY: bin
 bin: ## build the odo binary
+	cp ododevapispec.yaml pkg/apiserver-impl/swagger-ui/swagger.yaml
```

Comment:
guess we should improve the [`check-generated-files` workflow](https://github.com/redhat-developer/odo/blob/main/.github/workflows/check-generated-files.yaml) to make sure `pkg/apiserver-impl/swagger-ui/swagger.yaml` in kept in sync with `ododevapispec.yaml`.

Reply:
As this file is copied every time we build odo, I would say no (it is different from the ui files which are much more longer to generate/copy and which we need to copy manually)

Reply:
Yes, but currently, it is copied every time you build with `make bin`, but not every time you run `make install` (which I often do instead of `make bin`)..

Reply:
And even if it is copied, if the developer chooses not to commit/push the `pkg/apiserver-impl/swagger-ui/swagger.yaml` file, we might end up with a binary not serving the right spec file. So we need to take care of this when reviewing PRs changing the OpenAPI spec file..